### PR TITLE
Tile hover states

### PR DESCRIPTION
**Pull request by codeswan**
Adds the option for a `Tile` to have a background color or border on hover.

### CODE REVIEW

File: src/js/components/Tile.js [hunk 1]
```diff
@@ -43,11 +44,15 @@ export default class Tile extends Component {
 Tile.propTypes = {
   selected: PropTypes.bool,
   wide: PropTypes.bool,
+  hoverStyle: PropTypes.string,
```
Comment (ericsoderberghp) [inline src/js/components/Tile.js]: Is the style free-form or can we use PropTypes.oneOf()?

**Comment by ericsoderberghp** [comment]
What happens if the tiles don't have an `onClick` property but do have the hover properties set?

File: src/js/components/Tile.js (hunk 1 above)
Comment (codeswan) [inline src/js/components/Tile.js]: Good point, I've updated that.

**Comment by codeswan** [comment]
@ericsoderberghp I think that would be okay.  I've updated it so the default is `'none'` for the `hoverStyle`, so I think it's a safe assumption that if a user does set a `hoverStyle`, then they want it to show regardless of whether the `Tile` has an onClick or not.  What do you think?  I can change this if needed.

**Pull request merged**